app/assets/javascripts/components/activity/activity_table_row.jsx

```diff
@@ -65,7 +65,9 @@ const ActivityTableRow = React.createClass({
 
     let diffViewer;
     if (this.props.revision && this.props.revision.api_url) {
-      diffViewer = <DiffViewer revision={this.props.revision} />;
+      diffViewer = <DiffViewer revision={this.props.revision}
```

Comment:
For a single revision yes, but I did not see that available for rev / first rev usage - so I added the prop and used it uniformly throughout (even though for single rev I could get it from the revision itself).

(please check me on that - I may have missed something re two-revision usage)